frontend/src/employee-frontend/components/child-information/assistance-need/decision/AssistanceNeedDecisionEditPage.tsx

```diff
@@ -177,35 +238,43 @@ export default React.memo(function AssistanceNeedDecisionEditPage() {
       </Content>
       <Gap size="m" />
       <StickyFooter>
-        {showFormErrors && missingFields.length > 0 && (
-          <>
-            <H2>
-              {i18n.childInformation.assistanceNeedDecision.validation.title}
-            </H2>
-            <AlertBox
-              message={
-                <>
-                  <P noMargin>
-                    {
-                      i18n.childInformation.assistanceNeedDecision.validation
-                        .description
-                    }
-                  </P>
-                  <ul>
-                    {missingFields.map((field) => (
-                      <li key={field}>
-                        {field in i18n.childInformation.assistanceNeedDecision
-                          ? i18n.childInformation.assistanceNeedDecision[field]
-                          : field}
-                      </li>
-                    ))}
-                  </ul>
-                </>
-              }
-            />
-            <HorizontalLineWithoutBottomMargin slim />
-          </>
-        )}
+        {showFormErrors &&
+          Object.entries(fieldInfos).filter(([_, value]) => value).length >
```

Comment:
You can omit the `_` variable name completely and use `.filter(([, value]) => value)` instead.